content/en/data_jobs/databricks.md

```diff
@@ -19,34 +19,38 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
 
 1. [Configure the Datadog-Databricks integration](#configure-the-datadog-databricks-integration) with your Databricks API token.
 1. [Install the Datadog Agent](#install-the-datadog-agent-on-your-databricks-clusters) on your Databricks cluster(s).
-1. [Add your Datadog API key](#add-your-datadog-api-key-in-databricks) in Databricks.
 
-### Configure the Datadog-Databricks integration 
+### Configure the Datadog-Databricks integration
 
 1. In your Databricks workspace, go to **Settings** > **Developer**. Next to **Access tokens**, click **Manage**.
 1. Click **Generate new token**, enter a comment, and click **Generate**. Take note of your token.
 1. In Datadog, open the Databricks integration tile.
-1. On the **Configure** tab, click **Add New**. 
+1. On the **Configure** tab, click **Add New**.
 1. Enter a workspace name, your Databricks workspace URL, and the Databricks token you generated.
    {{< img src="data_jobs/databricks/configure-token.png" alt="In the Datadog-Databricks integration tile, a Databricks workspace is displayed. This workspace has a name, URL, and API token." style="width:100%;" >}}
 
 
 ### Install the Datadog Agent on your Databricks cluster(s)
 
-Use one of the following init scripts:
-- [Install the Datadog Agent][8]
-- [Install the Datadog Agent without logs][2]
+The following init script will install the Datadog Agent on the cluster:
+- [`datadog_databricks_job_monitoring_init.sh`][8]
 
 You can choose to install the Agent globally, or on a specific Databricks cluster.
 
 {{< tabs >}}
 {{% tab "Global init (Recommended)" %}}
-1. In Databricks, go to **Settings** > **Compute**. In the **All purpose clusters** section, next to **Global init scripts**, click **Manage**.
+1. In Databricks, go to **Admin Settings** > **Compute**. In the **All purpose clusters** section, next to **Global init scripts**, click **Manage**.
 1. Click **Add**. Name your script. Then, in the **Script** field, copy and paste the init script.
 1. To enable the script for all new and restarted clusters, toggle **Enabled**.
    {{< img src="data_jobs/databricks/toggle.png" alt="Databricks UI, admin settings, global init scripts. A script called 'install-datadog-agent' is in a list with an enabled toggle." style="width:100%;" >}}
 1. Click **Add**.
 
+#### Add your Datadog API key in Databricks
```

Comment:
It was confusing reading the header for this section as "Add your Datadog API key in Databricks" and separately "set the init script parameters". These are really just one step: "Set the required init script parameters", of which API key is one of them.

Also
1. For API key specifically, we should link to /organization-settings/api-keys to tell them they can go there in app to get this or create a dedicated on.
2. We should make a callout to make sure the DATABRICKS_WORKSPACE matches the value set up in the config for the Databricks Integration Tile for that namespace.